### PR TITLE
added docs for multichain

### PR DESCRIPTION
**Pull request by Saviour1001**

### CODE REVIEW

**Review by shahbaz17 — CHANGES_REQUESTED** [review]
Please make the changes as discussed over the call.

File: docs/connect-blockchain/multi-chain.mdx [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+  SolanaPrivateKeyProvider,
+  SolanaWallet,
+} from "@web3auth/solana-provider";
+
+/*
+  Use code from the above Initializing Provider here
+*/
+
+const privateKey = provider.request({ method: "eth_privateKey" });
+const { getED25519Key } = await import("@toruslabs/openlogin-ed25519");
```
Comment (shahbaz17) [inline docs/connect-blockchain/multi-chain.mdx:287]: @Saviour1001 please change this to use the function from `@web3auth/solana-provider` package.

**Review by shahbaz17 — CHANGES_REQUESTED** [review]

**Comment by shahbaz17** [comment]
Hey @Saviour1001 

Can you please look at https://web3auth.io/docs/sdk/web/providers/solana#additional-reading? And check if this has any impact on this PR.

**Comment by Saviour1001** [comment]
Got it changed !

**Review by shahbaz17 — APPROVED** [review]
LGTM

**Pull request merged**